test-app/tests/unit/setup-context-test.js

```diff
@@ -29,12 +29,14 @@ import config from '../../config/environment';
 import Ember from 'ember';
 import { deprecate, warn } from '@ember/debug';
 
+console.log('1', Ember.testing);
```

Comment:
seems console logs could be omitted here

Reply:
Oops. Thanks!